code/processes/wdb.q

```diff
@@ -166,7 +166,7 @@ savetablesbypart:{[dir;pt;forcesave;tablename;writedownmode]
 savetables:$[writedownmode in partwritemodes;savetablesbypart[;;;;writedownmode];savetables];
 
 savetodisk:{[]
-    changes:savetables[savedir;getpartition[];0b;] each tablelist[];
+    changes:savetables[savedir;getpartition[];1b;] each tablelist[];
```

Comment:
I don't think we _always_ want to have immediate writing enabled, this should be configurable

Reply:
Updated and reran tests